src/modules/renderList.js

```diff
@@ -0,0 +1,60 @@
+import ListElement from './listElement.js';
+
+export default class RenderList {
+  constructor() {
+    this.toDoList = localStorage.getItem('toDoList')
+      ? JSON.parse(localStorage.getItem('toDoList'))
+      : [];
+    this.addTask = this.addTask.bind(this);
+    this.renderList();
+  }
+
+  addTask(description) {
+    const task = new ListElement(this.toDoList.length + 1, description, false);
+    this.toDoList.push(task);
+    this.renderList();
+    this.toLocalStorage();
+  }
+
+  renderList() {
+    const listContainer = document.querySelector('.list__container__checklist');
+    listContainer.innerHTML = '';
+
+    // eslint-disable-next-line no-restricted-syntax
```

Comment:
- [ ] Like the previous CR mentioned please avoid disabling linters, as they are used to make your code better and easier to read. if something is not clear or if you have a question please leave a comment in the PR.

Reply:
Oh, man, I forgot to delete the comment. So, I will solve this right now.